operator/controllers/lifecycle/keptnevaluation/controller.go

```diff
@@ -264,10 +264,16 @@ func (r *KeptnEvaluationReconciler) fetchDefinitionAndProvider(ctx context.Conte
 	if err := r.Client.Get(ctx, namespacedDefinition, evaluationDefinition); err != nil {
 		return nil, nil, err
 	}
+
```

Comment:
also I know we have no unit tests for this guy, but maybe you could add it at least for this function, to show it some love?